app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- You have done great so far, but there are a few requirements you still need to meet before moving to the next milestone.

- Screenshot reference:
![Selection_048](https://github.com/stino-x/Blog-Rails-/assets/107632780/fd715639-33f9-4c2e-97f8-0487234f431e)

- [ ] As stated in the project requirement, newly registered users need to confirm their email, and for this to work, you need to enable `:confirmable` in our `user model`. I suggest you do this. Also remember to un-comment this part of the code in your devise migration file for this to work correctly. :+1: :smile: 

```ruby
     t.string   :confirmation_token
      t.datetime :confirmed_at
      t.datetime :confirmation_sent_at
      t.string   :unconfirmed_email # Only if using reconfirmable
```

- [ ] Also I noticed that in your Application controller, you still have the `current_user`, according to the requirement, you are meant to remove this as Devise provides a method to do this by default. :+1: :100: